wordpress-mobile/WordPress-Android
# Issue/8479 add analytics to revisions

Fixes #8479

This PR adds analytics to track Revisions functionality. 
What we are tracking:

1. Opening revision list.
2. Opening revision detail from revision list.
3. Navigating to revision detail using chevrons at the bottom of the detail screen.
4. Navigating to revision detail using swipe.
5. Closing revision detail using either close button (X) or back button.
6. Loading revision.
7. Undo loading of revision.

Test

1. Add breakpoints to the lines with the tracker code.
2. Make sure they are hit when expected.

([iOS issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/10309))
